@xen-orchestra/lite/src/locales/fr.json

```diff
@@ -29,12 +26,32 @@
   "export": "Exporter",
   "export-table-to": "Exporter le tableau en {type}",
   "export-vms": "Exporter les VMs",
-  "hosts": "Hôtes",
-  "language": "Langue",
-  "last-week": "Semaine dernière",
+  "filter": {
+    "comparison": {
+      "contains": "contient",
+      "does": "",
+      "does-not": "ne",
+      "ends-with": "termine par",
+      "equals": "est égal à",
+      "is": "est",
+      "is-false": "est faux",
+      "is-not": "n'est pas",
+      "is-true": "est vrai",
+      "matches-regex": "correspond à la regex",
+      "not-contain": "ne contient pas",
+      "not-end-with": "ne termine pas par",
+      "not-equal": "n'est pas égal à",
+      "not-match-regex": "ne correspond pas à la regex",
+      "not-start-with": "ne commence pas par",
+      "starts-with": "commence par"
```

Comment:
IMO, it must start with a capital letter and modified in CSS / JS.

Reply:
I don't think there are any pros/cons. It's just to have something homogeneous and not have to search if the message starts with a capital letter or not when we want to reuse it.